install_rock.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-if [ "$(uname -o)" == "Msys" ]; then
-	tmp="$(wget -O - https://github.com/cogneco/rock/releases/latest | grep -o '\<download/rock_.*zip\>')"
-	wget https://github.com/cogneco/rock/releases/$tmp
-	echo ${tmp##*/}
-	7z e ${tmp##*/}
-else
```

Comment:
As far as the script goes, it's all versioned, so we can revive if need be (although if the file is moved or renamed again, it'll be harder to find this part of it).